data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -327,6 +329,7 @@
         MeasurementRecordsSuppression,),
     (CleanHeightAndWeight,),  # dependent on MeasurementRecordsSuppression
     (UnitNormalization,),  # dependent on CleanHeightAndWeight
+    (CreateDerivedTables,),
```

Comment:
I would suggest moving this lower after concept id cleaning, maybe just before CalculatePrimaryDeathRecord